Ryujinx.HLE/HOS/Services/Hid/IHidServer.cs

```diff
@@ -574,6 +576,20 @@ public ResultCode IsSixAxisSensorAtRest(ServiceCtx context)
             return ResultCode.Success;
         }
 
+        [CommandHipc(83)] // 6.0.0+
+        // IsFirmwareUpdateAvailableForSixAxisSensor(nn::hid::SixAxisSensorHandle, nn::hid::AppletResourceUserId, pid) -> bool UpdateAvailable
+        public ResultCode IsFirmwareUpdateAvailableForSixAxisSensor(ServiceCtx context)
+        {
+            int  sixAxisSensorHandle  = context.RequestData.ReadInt32();
```

Comment:
you need to skip 4 bytes because of padding
```suggestion
            int  sixAxisSensorHandle  = context.RequestData.ReadInt32();
            context.RequestData.BaseStream.Position += 4;
```

Reply:
Thanks for confirming my suspicion! I noticed in testing that padding was probably necessary to get accurate numbers, but since all other calls dont have this padding I decided against adding it. Should I apply this padding to the other commands which don't currently have it?

Reply:
It really depends of the offset at which a field is placed and the next field size.

Basically this follow standard C alignment *most* of the time.

example 1 (padding):
```c
struct A {
    int  field_0x0;
                    // padding of 4 bytes right after field_0x0 to meet the alignment requirements for field_0x8. (sizeof(long) - sizeof(int) = 4)
    long field_0x8;
}
```

example 2 (no padding):
```c
struct A {
    int  field_0x0;
    int  field_0x4;
                    // no padding after field_0x4 because it meet the alignment requirements for field_0x8. (sizeof(long) - (sizeof(int) + sizeof(int)) = 0)
    long field_0x8;
}
```

Reply:
Alright, so just add the sizes of all previous fields together and check how many padding bytes to add to get to a multiple of 8 bytes, correct? 

Adding the same padding to IsSixAxisSensorAtRest and ResetSixAxisSensorFusionParameters (among others) fixes their applet ids as well, so I can confirm that this would fix other commands as well. Should I open up another PR for those fixes, or just add them to this one? (Personally I'd open up another PR, to not clutter up this one with unrelevant changes)

Reply:
Another PR would be better 👍

Reply:
Great! I'll do that then.